### DISCUSSION TITLE
Have Renovate not assign a reviewer who is already assigned to an open Renovate PR

### DISCUSSION THREAD

**Discussion by flangeratlassian** [renovatebot/renovate#22201] [Suggest an Idea]
**What would you like Renovate to be able to do?**

If a reviewer is already assigned to an open Renovate PR and there are others who are not assigned to any, select from them instead. If all possible reviewers are already assigned to open Renovate PRs, allow repeats.

**Did you already have any implementation ideas?**

I haven't explored the code yet

**Comment by rarkins** [comment]
I'm not sure if that can be done elegantly. For assignees we have https://docs.renovatebot.com/configuration-options/#assigneessamplesize which could achieve similar aims

**Comment by flangeratlassian** [comment]
I did have sample size configured as well. Unfortunately it doesn't prevent a reviewer from being picked again. I was thinking the mechanism I'm describing sounds most similar to https://docs.renovatebot.com/configuration-options/#prconcurrentlimit. Could have a property called `assigneeConcurrentLimit` that would limit how many open PRs a person can be assigned to. Then Renovate can throw an error when creating a PR to which it has no more people to assign. I've seen it raises a similar error when there is an invalid team-member user id. Of course these details are just an idea; perhaps they make you think of an existing mechanism with which to do this elegantly, though?

**Comment by rarkins** [comment]
So far this feature seems like it would like require a lot of complexity but for only marginal benefit. If you can step through the code base and try to work out *if* there's an elegant way to implement it then we can consider it.
